Packages/com.unity.inputsystem/InputSystem/Plugins/OnScreen/OnScreenButton.cs

```diff
@@ -1,3 +1,4 @@
+#if UNITY_INPUT_SYSTEM_ENABLE_UI
```

Comment:
I realized we need to add `|| PACKAGE_DOCS_GENERATION` to the #ifs that have public APIs in the files.

Reply:
That would explain my failing test